src/index.ts

```diff
@@ -37,3 +37,24 @@ export const formatDate = ({
   if (isYesterday(dateValue)) return yesterdayText;
   return date;
 };
+
+export const formatTime = (time_in_seconds: number) => {
```

Comment:
@aswindevps 

1. The site is broken as there is no documentation for this method.
2. Can you please use camelcase instead of snake case?